app/server/src/server/build.ts

```diff
@@ -0,0 +1,4 @@
+import { buildStatic } from '@storybook/core/server';
+import options from './options';
+
+buildStatic(options);
```

Comment:
What do these files do? Were they copied over from `@storybook/html`?

Reply:
yup copied. I'm fuzzy on what this stuff is needed for. Should we nuke?

Reply:
Do we need this at all?

Reply:
I think we need this:

```
  "bin": {
    "build-storybook": "./bin/build.js",
```